core/server/api/v2/redirects.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs-extra');
+const path = require('path');
+const Promise = require('bluebird');
+const moment = require('moment-timezone');
+const config = require('../../config');
+const common = require('../../lib/common');
+const validation = require('../../data/validation');
+const web = require('../../web');
+
+const _private = {};
+
+_private.readRedirectsFile = (customRedirectsPath) => {
+    const redirectsPath = customRedirectsPath || path.join(config.getContentPath('data'), 'redirects.json');
+
+    return fs.readFile(redirectsPath, 'utf-8')
+        .then((content) => {
+            try {
+                content = JSON.parse(content);
+            } catch (err) {
+                throw new common.errors.BadRequestError({
+                    message: common.i18n.t('errors.general.jsonParse', {context: err.message})
+                });
+            }
+
+            return content;
+        })
+        .catch((err) => {
+            if (err.code === 'ENOENT') {
+                return Promise.resolve([]);
+            }
+
+            if (common.errors.utils.isIgnitionError(err)) {
+                throw err;
+            }
+
+            throw new common.errors.NotFoundError({
+                err: err
+            });
+        });
+};
+
+module.exports = {
+    docName: 'redirects',
+
+    download: {
+        headers: {
+            disposition: {
+                type: 'file',
+                value: 'redirects.json'
+            }
+        },
+        permissions: ['redirects', 'download'],
```

Comment:
Is this notation correct? 🤔 
Shouldn't it be docName and method?

Reply:
Yes, you are completely right, this notation is excessive. Just `permissions: true` does it :)